Lombiq.NodeJs.Extensions/scripts/handle-error.js

```diff
@@ -46,6 +60,25 @@ function handleErrorObjectInner(error, type, defaultCode) {
     return error;
 }
 
+function handleErrorObjectForGitHub(type, code, message, path, line, column) {
+    const parameters = ['title=' + code];
+    const gitHubRoot = process.env.LOMBIQ_NODEJS_EXTENSIONS_GITHUB_ROOT;
+
+    if (path) {
```

Comment:
Is this most recent failing run how an error looks like currently too? https://github.com/OrchardCMS/OrchardCore.Commerce/actions/runs/14615008579/job/41008923526 Because the error messages don't show the file and line number.

Reply:
Very strange behavior, the error annotations are not created in the summary. I have added the path to the annotation message, so it's always available.

Reply:
Here is a newer example of the lint errors: https://github.com/Lombiq/Orchard-Privacy/actions/runs/14820609336/job/41607027094

Reply:
Looks good!